config/configuration.json

```diff
@@ -297,6 +297,19 @@
       "tags":"highway"
     }
   },
+  "RoundaboutConnectorCheck": {
+    "threshold": {
+      "one_way": 100.0,
+      "two_way": 130.0
+    },
+    "challenge": {
+      "description": "Roads connected to roundabouts at too sharp an angle.",
+      "blurb": "Sharp intersections between roundabouts and connecting roads are often digitization/navigation errors. Adjust these intersections to create smoother transitions in an out of roundabouts.",
```

Comment:
typo: "Adjust these intersections to create smoother transitions in ~an~ **and** out of roundabouts."